docs/getting_started.rst

```diff
@@ -94,13 +126,14 @@ We can also convert ``PauliString`` objects back to dense numpy arrays if we'd l
     # Returns "XYZ"
```

Comment:
I think similar `# Returns` comment needed to describe the output of `P_np = P.to_tensor()`. because it's easy to confuse it with `# Returns "XYZ"`